packages/cli/src/build.ts

```diff
@@ -121,16 +121,27 @@ export async function build({
             if (!indexFile && outputSources && filePath.startsWith(fullOutDir)) {
                 return false;
             }
+            // assets changes that tracked by stylable file should retrigger processFiles
+            if (assets.has(filePath)) {
+                return true;
+            }
+            // stylable files
             return filePath.endsWith(extension);
         },
         onError(error) {
             console.error(error);
         },
         processFiles(service, affectedFiles, deletedFiles, changeOrigin) {
             if (changeOrigin) {
+                // watched file changed invalidate cache
                 stylable.initCache();
+                // handle deleted files by removeing thier generated content
                 if (deletedFiles.size) {
                     for (const deletedFile of deletedFiles) {
+                        if (assets.has(deletedFile)) {
+                            assets.delete(deletedFile);
+                            continue;
+                        }
```

Comment:
`else if` for line below?

Reply:
no need because if the continue...but will do.